freetextresponse/tests.py

```diff
@@ -260,6 +261,23 @@ def test_compute_score(self, credit):
             {'value': credit.value, 'max_value': Credit.full.value},
         )
 
+    @ddt.data(Credit.zero, Credit.half, Credit.full)
```

Comment:
We should not need to run this test for each type of credit to verify the exception is handled properly.   We still need to mock determine_credit however, so just use any one of the credit values.